buildkite/scripts/fuzzy-zkapp-test.sh

```diff
@@ -9,7 +9,8 @@ fi
 
 profile=$1
 path=$2
-trials=$3
+timeout=$3
+individual_test_timeout=$4
```

Comment:
You need to change the `3` on line 5 to a `4` for this to work. Usage description should also change.